### DISCUSSION TITLE
Carbon Fiber frame for Bolt 1.1

### DISCUSSION THREAD

**Discussion by simondlevy** [bitcraze/discussions#847] [Ideas]
I just discovered that a colleague here has a waterjet cutting machine I can use, and I'm thinking of what @tobbeanton said in his workshop [video ](https://www.youtube.com/watch?v=xiWLhr-HpG8)about  brushless builds -- it would be nice to have a carbon-fiber frame custom-designed for the Bolt!  The 122mm [frame ](https://www.65drones.com/cdn/shop/products/20161106_131826.jpg
) used in some previous builds is no longer being sold, probably because all the newer micro frames use the 20x20mm mounting pattern.   Is there still any interest at Bitcraze in doing this?  I'd be happy to contribute by prototyping some frame designs, or waterjet-cutting ones that you guys may have in mind.

**Comment by tobbeanton** [comment]
A carbon-fiber frame would definitely be interesting. I've seen some prototypes that clients did using the [Li-Ion frame](https://www.bitcraze.io/2020/10/testing-crazyflie-bolt-and-1-cell-li-ion/) as base. I think the size that frame is very nice but it could definitely be improved! Please give it a shot!

**Comment by simondlevy** [comment]
![20230801_200118](https://github.com/bitcraze/discussions/assets/6766537/9661d4e9-ead1-46cb-b9a8-4723437d8957)
Ugly 3D-printed prototype, 'cuz the tech-support guy is on vacation, LOL!

**Reply by tobbeanton** [reply]
Interesting print! What props and motors are you designing for, it looks compact which will effect efficiency negatively. But if size is of importance it can be worth that efficiency hit. Otherwise I would recommend 3" props for the Bolt.

**Comment by simondlevy** [comment]
That's right, I am specifically aiming for no more than 100mm "wheelbase" (diagonal motor-to-motor distance), and no more than 100g "dry weight" (sans battery), for the research paper I am working on. (My guiding principle is "tiny neuromorphic flight control")  I still plan to use the 18650 LiIon battery and ~11,000kV 1s motors from your workshop [video](https://youtu.be/xiWLhr-HpG8?t=949), which I've already [flown](https://www.youtube.com/watch?v=HcN851RfTG8) successfully on your frame design.  But for this new carbon-fiber design I will have to use [two-inch props](https://www.getfpv.com/propellers/micro-quad-propellers/gemfan-2023-3-blade-2-propellers-for-mobula8-set-of-4-clear-gray.html) -- while lamenting that we still cannot get rid of Imperial Weights and Measures! ;^D

**Reply by simondlevy** [reply]
Update: I ended up going with [2.5-inch props](https://www.racedayquads.com/products/gemfan-d63-ducted-durable-tri-blade-63mm-prop-8-pack) and [850mAH 1S lipo](https://www.racedayquads.com/products/rdq-series-3-8v-1s-850mah-60c-lihv-whoop-micro-battery-ph2-0).

**Comment by simondlevy** [comment]
![264764447-85b5c88c-6d81-4fa3-b7fb-179c334246ab](https://github.com/bitcraze/discussions/assets/6766537/a72601d7-767b-4c65-b862-7c3b44570d2d)
Carbon-fiber frame top and bottom.

**Reply by CalvinKoung** [reply]
Hi, Are you willing to release the CAD file? or stl ?

**Reply by simondlevy** [reply]
![FrameBottom](https://github.com/bitcraze/discussions/assets/6766537/d3a89e5a-84e7-48c8-91a8-a58a0bb11500)
![FrameTop](https://github.com/bitcraze/discussions/assets/6766537/fd613b11-5ee8-499e-97fb-a77d2a1ad11d)

**Reply by simondlevy** [reply]
I've attempted to upload the SVG files; let me know if you can't access them.  I can also share STL, but the SVG files were the result of some cleanup by our technician, so probably better if you want to do the laser-cut carbon fiber.

**Reply by CalvinKoung** [reply]
You right I can't access the SVG files.

**Reply by simondlevy** [reply]
Sorry, I've had some difficulty recovering the original files.  I've made (and attempted to attach) a zipfile containing the frame bottom as an SVG and the top as an STL.
[Bolt_1_1_frame.zip](https://github.com/bitcraze/discussions/files/15050719/Bolt_1_1_frame.zip)

**Comment by simondlevy** [comment]
![265841223-9785f68c-13c2-4979-bcb3-73f5157adf41](https://github.com/bitcraze/discussions/assets/6766537/8ea779c1-cea9-483b-a98c-7d316c8af609)
Frame with Bolt 1.1 and motors, top view.

**Comment by simondlevy** [comment]
![265841304-d67bd8ed-69a2-43a9-88da-827b31312834](https://github.com/bitcraze/discussions/assets/6766537/f9fb7d09-9320-4179-8018-0504893a211a)
Bottom view, showing[ JST PH2.0 battery connector](https://www.amazon.com/gp/product/B088NFRNZ2) replacing the XT30 connector that comes with the Bolt 1.1. Sloppy soldering partially hidden by [Liquid Tape](https://www.amazon.com/Gardner-Bender-07315001126-LTB-400-Electrical/dp/B000FPAN2K) ;^)

As soon as my Flow Decks arrive, it'll be time for a maiden flight!

**Reply by tobbeanton** [reply]
Progress, very nice! Looking forward to pictures of the full assembly :wink:

**Reply by knmcguire** [reply]
and the video of the maiden flight ofcourse!

**Comment by simondlevy** [comment]
https://www.youtube.com/watch?v=0On86N23ibc

At some point I've gotta learn how to land, LOL!

**Comment by simondlevy** [comment]
![20230912_204849](https://github.com/bitcraze/discussions/assets/6766537/46fca198-d039-4f0d-b778-4a7f311bdca4)
![20230912_204859](https://github.com/bitcraze/discussions/assets/6766537/d98156b4-8370-4e49-accb-78eec40fb463)
![20230912_204958](https://github.com/bitcraze/discussions/assets/6766537/f3724640-a9f7-4abe-b9be-6037ffbbc18d)
I've got 2" props on it, but I can probably squeeze in 2.5" for more lift.

**Reply by tobbeanton** [reply]
That's some power! Be careful :wink:

**Reply by simondlevy** [reply]
"Go brushless or go home!" LOL
